README.md

```diff
@@ -1,3 +1,135 @@
-# wrench
+# Wrench
 
 [![Travis](https://img.shields.io/travis/tomologic/wrench.svg?style=flat-square)](https://travis-ci.org/tomologic/wrench)
+
+## What is this?
+
+This is our current try to create standards for building, testing and deploying our applications.
+
+What we want:
+- General pipelines that can build, test and deploy all our apps/services
+- Useful development tooling for local development
+
+Putting the logic in the app repos would create a lot of duplications, big overhead when creating new services and maintenance hell. Putting the logic in the pipelines will make local development difficult since all logic for building/testing/deploying exists in the CI systems. This implies that logic for building/testing/deploying apps/services need to be accessible for both developers and CI system. 
```

Comment:
"creating new services and maintenance hell" could be reshuffled to not imply that the user gets overhead when creating maintenance hell

"logic [...] needs* to be accessible"